src/phoenix/db/models.py

```diff
@@ -120,7 +125,15 @@ class Trace(Base):
     trace_id: Mapped[str]
     start_time: Mapped[datetime] = mapped_column(UtcTimeStamp, index=True)
     end_time: Mapped[datetime] = mapped_column(UtcTimeStamp)
-    latency_ms: Mapped[float]
+
+    @hybrid_property
+    def latency_ms(self) -> float:
+        return (self.end_time - self.start_time).total_seconds() * 1000
```

Comment:
Nice, I dig it.

Reply:
the `.inplace.expression` part below this does that

Reply:
basically, `classmethod` are db operations, while instance methods are python operations